community/posters.md

```diff
@@ -0,0 +1,59 @@
+# Event Poster Guidelines
+See also: The [design](https://github.com/kossiitkgp/design/) repository. Please add all poster designs to this repository.
+
+### Poster Design Guidelines
+#### Size
+- Instagram restricts images in a post to 1:1 ratio.
+- For printable posters, the size must be **11.5x17.5 inches** and it must be in portrait orientation.
```

Comment:
Mention instagram wala part later, otherwise someone might just read this and create 1:1 poster which won't be fit to be printed.

Reply:
The canvas size when you design should be 11.5x17.5 inches which is necessary for the poster to be printed properly.. this is must to mention.